theme/less/_body-wrapper.less

```diff
@@ -1,6 +1,6 @@
 body
 {
-  font-family: "PT Sans", "HelveticaNeue", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-family: "Source Sans Pro";
```

Comment:
Let's still specify fallback fonts, because if there is a network issue we might be able to use a local font, as it's a font from serif family, keeping `sans-serif ` as fallback or `Helvetica, Arial, sans-serif ` as it was before should be good to go. So the later one means, if `Source Sans Pro ` can't resolved for the character use  Helvetica, if no Helvetica on the system use Arial, if no Arial, then resolve to a sans-serif font.